packages/localforage/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@jupyterlite/localforage",
```

Comment:
Do we want to follow the name of the underlying library, or use something a bit more generic like `@jupyterlite/persistence`?

Reply:
But yeah, you know me: in theory i wish it could be generic! I think the larger refactor of #315 can address some more of these issues, but trying to abstract our storage layer at this point feels... premature. This gets us to a "doesn't fail by default unless you have insecure iframe settings," which seems like progress, and allows for at least integrating with more localforage options.

It's kinda like codemirror: it's really good at what it does (e.g. fallback) but has a lot of global state in The Singleton `localforage`. Even with `sharedPackages`, etc. weird stuff can still happen, so i wanted it to be very explicit. 

If one wants a new storage driver, there are _way_ more localforage examples to look at and emulate than pretty much our _one example_ of contents and settings persistence, so encouraging hooking that by example is the best I figured we can do (even if i didn't write the driver).

As present, one would have to pretty much replace the `IContents` and `ISettings` providers wholesale to change the behavior much, otherwise (e.g. use a new or existing REST server), and as such those _don't_ have `localforage` anywhere on their boundaries, and only the routes consume _those_, so i think we're covered. Even then, maybe a localForage API is useful in its own right.